packages/next/src/server/app-render/make-get-server-inserted-html.tsx

```diff
@@ -11,14 +11,6 @@ import { RedirectStatusCode } from '../../client/components/redirect-status-code
 import { addPathPrefix } from '../../shared/lib/router/utils/add-path-prefix'
 import type { ClientTraceDataEntry } from '../lib/trace/tracer'
 
-export function getTracedMetadata(
```

Comment:
Moved this to the trace utils as a generic place because here it was in the app router code and we also want to use it in the pages router.